examples/decorator/cargo-generate.toml

```
@@ -1,5 +1,5 @@
 [template]
-ignore = [".tox"]
+ignore = [".tox", ".nox"]
```

```suggestion
ignore = [".nox"]
```

and similar for the other files